### DISCUSSION TITLE
ActivityWatch is an app that automatically tracks how you spend time on your devices.

### DISCUSSION THREAD

**Discussion by 391a** [community-scripts/ProxmoxVE#970] [Request script]
### Application Name

ActivityWatch

### Website

https://activitywatch.net/

### Description

The best free and open-source automated time tracker. Cross-platform, extensible, privacy-focused.
https://github.com/ActivityWatch/activitywatch
https://docs.activitywatch.net/en/latest/introduction.html

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by alexanderjerome** [comment]
One of the issues with activity watch is that it is meant to run only on the host that is using it - there really isn't an official headless server that can allow you to sync activity from multiple servers. It's [technically possible but not officially supported](https://docs.activitywatch.net/en/latest/remote-server.html).

In the past I've used [ManicTime](https://www.manictime.com/) which fulfilled that need, but its not open source or free. An open source and free alternative for automated time tracking based on activity would be so welcome, though, if anybody knows of an equivalent application.
